reflex/style.py

```diff
@@ -35,6 +35,66 @@
     _var_data=color_mode_var_data,
 )
 
+breakpoints = ["0", "30em", "48em", "62em", "80em", "96em"]
+
+
+def media_query(breakpoint_index: int):
+    """Create a media query selector.
+
+    Args:
+        breakpoint_index: The index of the breakpoint to use.
+
+    Returns:
+        The media query selector used as a key in emotion css dict.
+    """
+    return f"@media screen and (min-width: {breakpoints[breakpoint_index]})"
+
+
+def convert_item(style_item: str | Var) -> tuple[str, VarData | None]:
+    """Format a single value in a style dictionary.
+
+    Args:
+        style_item: The style item to format.
+
+    Returns:
+        The formatted style item and any associated VarData.
+    """
+    if isinstance(style_item, Var):
+        # If the value is a Var, extract the var_data and cast as str.
+        return str(style_item), style_item._var_data
+
+    # Otherwise, convert to Var to collapse VarData encoded in f-string.
+    new_var = Var.create(style_item)
+    if new_var is not None and new_var._var_data:
+        # The wrapped backtick is used to identify the Var for interpolation.
+        return f"`{str(new_var)}`", new_var._var_data
+
+    return style_item, None
```

Comment:
If `style_item` is not a `str` would the type signature still be correct here? Should it be `str(style_item`)?